RPIPs/RPIP-53.md

```diff
@@ -0,0 +1,110 @@
+
+
+## Motivation
+
+The Redstone update introduced the Merkle-tree-based rewards system, which allowed users to claim RPL they've earned for both node operating duties and Oracle DAO duties, as well as their share of Smoothing Pool rewards. Since its inception, this system has gone through several revisions to account for fixes and Ethereum upgrades but has fundamentally remained the same: the tree lists rewards earned for individual node operators, and node operators can claim those rewards at any time.
+
+On that note, Houston (or more specifically RPIP-31) has an unfortunate consequence: if you have an RPL withdrawal address set, [you can no longer claim rewards intervals](https://github.com/rocket-pool/rocketpool/blob/a747457fc610aa889af0a13cff5d3d00955525a5/contracts/contract/rewards/RocketMerkleDistributorMainnet.sol#L85). This includes both RPL rewards and Smoothing Pool ETH rewards. Claiming them *must* be done from the RPL withdrawal address. Whether intended or not, the RPL withdrawal address isn't really an RPL withdrawal address in Houston; it's a "claimer address" that can claim rewards intervals from the Merkle distributor contract.
+
+This is problematic for a few reasons. As an example: the node is still going to be listed in the rewards file, and thus the rewards for the corresponding interval are still going to show up as a "claimable interval" in the Smart Node despite the user not being able to claim from it, and not being able to actually *receive* any of the rewards from that interval. It will only disappear from the list once the RPL withdrawal address has claimed from it. This can obviously be worked around in the Smart Node but it's exactly that: a workaround. The file itself is still going to have the node operator listed, so anyone (such as a third party) that wants to use it to view or claim rewards has to know about and implement this workaround as well.
+
+In this RPIP, I'm proposing a small change to the rewards file that ameliorates the problems and provides some helpful benefits for people that would actually take advantage of the feature in the first place.
+
+
+## Important Clarification
+
+Note that the RPL withdrawal address, as specified in RPIP-31, is **not** the same as the "primary withdrawal address" (previously known as just the "withdrawal address" in Atlas and below). If a node operator has a "regular" withdrawal address set, they will *not be affected by this change*. They can still claim rewards normally. The only nodes affected by this change are ones that explicitly set an **RPL withdrawal address**, which is a different setting than the "primary withdrawal address", after Houston has been deployed.
+
+
+## Specification
+
+The specification is going to be one of two possibilities:
+- Either an amendment to v8 [from the research repo]((https://github.com/rocket-pool/rocketpool-research/blob/master/Merkle%20Rewards%20System/merkle-tree-spec.md)), or
```

Comment:
This should presumably refer to RPIP-51 rather than externally.

Reply:
Yep, you're right - I will amend it to link to RPIP-51 instead. Thanks for the catch.